### ISSUE TITLE
Renamed Addon

### ISSUE THREAD

**Issue by JanWaller** [Luxocracy/TidyPlatesContinued#4]
Won't I lose all my settings due to the rename?

**Comment by Luxocracy** [comment]
Yes, due to the rename you would lose all of your settings.
I would use [this commit](https://github.com/Luxocracy/TidyPlates_BFA/tree/690bebab2923b976b1ed353c896f17fd92135479) if you want to avoid losing your settings for now.

And honestly I am not sure how I feel about the whole situation with this now _apparently_ becoming a standalone addon rather than a fix in the absence of the original authors.

**Comment by AHigi** [comment]
The authors aren't responding, however if you want the addon to be released via curseforge it needs a new name. 
The settings can be saved with a simple rename of the lua files in the wtf folder by replacing TidyPlates with TidyPlatesContinued
And after that edit the files and replace TidyPlates... with TidyPlatesCont.
About the addon being represented on curse, now it is actually not wanting to accept the uploaded zip. I contacted the support regarding this issue. Plus wrote a pm @Luxocracy on curse regarding a setting that he has to make

**Comment by kinsluck** [comment]
If the original author don't reply your requests, I think it's reasonable to start up a new project on curseforge.
When the author get back to the project, you can negotiate with him and merge the project to the origin.

**Comment by Luxocracy** [comment]
Yeah, if the authors won't respond I guess there is nothing else to do but start up a new one.

  I've fixed up the git so that any release here is getting pushed to Curse, however it still gets rejected and won't tell me why(I am assuming AHigi is getting the mail regarding it)

**Comment by AHigi** [comment]
Addon is now available on curse :)

**Comment by ja0b** [comment]
What was the issue @AHigi? And here is the beautiful link to it https://www.curseforge.com/wow/addons/tidy-plates-continued

**Comment by AHigi** [comment]
The issue is that if you upload the addon manually it always gets rejected... but with build hook it was just fine

**Comment by Luxocracy** [comment]
I mean it got rejected by the webhook as well, until you removed the classic plates. So I'd say that was the culprit.

**Comment by AHigi** [comment]
Not at all... At first I uploaded it without that one... And the system just returned that .pkgmeta was missing, ok so I included it in the zip. then it was the problem that it was in it.  The system is a bit... how should I say it strange to me.... It doesn't really specify the error. On the last build event there was the specific error that _classic was there but not the .toc file for that, until then it was just: "you did it wrong, here the documentation, gl figuring it out"
